### DISCUSSION TITLE
Werte der Liveansicht in Prometheus

### DISCUSSION THREAD

**Discussion by mastertape** [hoylabs/OpenDTU-OnBattery#642] [Q&A]
Ich suche vergeblich die daten die in der liveanischt bezüglich der gesamt erzeugung veräuche oder auch das was der stromzähler meldet bei den prometheus endpunkten oder durchblicke ich es noch nicjt richtig?
Gibt es alternativ die möglichkeit dies sonst übertelegraf /mqtt abzufragen?

**Comment by schlimmchen** [comment]
Die Prometheus-Integration wird von diesem Fork nicht ernsthaft unterstüzt. Das was vom Upstream kommt ist drin, aber Daten zum MPPT oder der Batterie etc. sind schlicht nicht implementiert.

**Reply by rudolfp1979** [reply]
Ist das geplant oder kann man mal was basteln und euch übergeben?

**Reply by schlimmchen** [reply]
Mir ist nicht bekannt, dass da jemand dran arbeitet. Gerne darfst du dich einbringen!

**Reply by stegbth** [reply]
Hi,
ist damit sowas gemeint?
![image](https://github.com/helgeerbe/OpenDTU-OnBattery/assets/168184161/b24ff598-2a7e-443e-97c8-0028a1b07f43)
oder sowas?
![image](https://github.com/helgeerbe/OpenDTU-OnBattery/assets/168184161/3f41e650-8753-4ca6-a817-ea50f554e155)

dieses Konstrukt habe ich mittels

1. mqtt - mosquitto
2. nodered
3. influxdb
4. grafana

mal zum Testen gebaut. Meines Wissens kann nodered die Daten ebenso an Prometheus anstatt Influxdb uebergeben.
Dabei bin ich nach zwei Tutorials im Internet vorgegangen.
Bei Bedarf kann ich die URLs nochmals raussuchen.

vg
Thomas

**Reply by rudolfp1979** [reply]
Nein ich meine z.B die Ausgabe auch von batteriestand und victron Daten ich werd mir die Tage mal ein esp32 Emulator an tun programmieren ist nicht das Problem.

**Reply by rudolfp1979** [reply]
hat sich zufällig schon wer nen wokwi config gebastelt auf der man aufbauen kann?

**Reply by stegbth** [reply]
Hi @rudolfp1979 ,

sind die von dir gewünschten Werte nicht in den MQTT Meldungen enthalten?
Auch wenn ich diese Graphen vom Shelly gemeldet weiterleite, sehe ich im MQTT-Explorer auch die Werte unter solar/powermeter.
Von daher vermute ich, dass die Werte vom Victron und der Batterie auch via MQTT gemeldet werden.

vg
Thomas

**Reply by rudolfp1979** [reply]
Ja nur verwende ich keinen mqtt Broker sonder rufe über Prometheus die api ab und die gibt diese Werte nicht aus soweit ich das übersehen kann
